notify.go

```diff
@@ -86,17 +86,23 @@ func notify(config *effectiveConfig, hook hookEvent, ws string, notification gri
 
 	if config.hipChatToken != "" && config.hipChatRoom != "" {
 		if err != nil {
-			logger.Printf("Hipchat: Error while rendering message: %v", err)
+			logger.Printf("Hipchat: Error while rendering message: %v\n", err)
 			return err
 		}
 
-		err = sendMessageToRoom(config.hipChatToken, config.hipChatRoom, config.grimServerID, message, color)
+		grimServerID := config.grimServerID
+		if len(grimServerID) > 15 {
+			grimServerID = fmt.Sprintf("%s", grimServerID[:15])
+			logger.Printf("grimServerID shouldn't be over 15 characters and has been truncated\nPlease update your config.json file to have a shorter grimServerID\n")
```

Comment:
grimServerID is the name of the in memory field.  In the config it is actually GrimServerID.  Also,  note it is possible to get the GrimServerID set by the "defaults" which is hte grim queue name, so they might not even have a GrimServerID.  Don't know what to do about that, maybe add to the error message?